expertsystem/ui/__init__.py

```diff
@@ -378,14 +376,12 @@ def find_solutions(
             )
 
         # merge strengths
-        final_result = Result[ParticleWithSpin]([])
+        final_result = Result([])
```

Comment:
Still don't really like that the caller has the responsibility of initializing an empty list for the `Result.solutions`.

Reply:
Thats true

Reply:
2d633a37a070d3397fb0bcddf633f28acb7239cc